api/src/main/java/uk/gov/dhsc/htbhf/hmrc/factory/EligibilityResponseFactory.java

```diff
@@ -1,33 +1,50 @@
 package uk.gov.dhsc.htbhf.hmrc.factory;
 
+import org.springframework.stereotype.Component;
 import uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus;
 import uk.gov.dhsc.htbhf.hmrc.entity.Child;
 import uk.gov.dhsc.htbhf.hmrc.entity.Household;
+import uk.gov.dhsc.htbhf.hmrc.model.ChildDTO;
 import uk.gov.dhsc.htbhf.hmrc.model.EligibilityResponse;
 
 import java.time.LocalDate;
+import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
 
+@Component
```

Comment:
Can we move this back to being a normal class with static methods? I see no reason why is should be Spring managed unless I'm mistaken.

Reply:
It felt like there was too much logic to not make it a dependency. The key for me was when the EligibilityServiceTest was failing because of an issue with the factory.